ballerina/Ballerina.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+org = "ballerinax"
+name = "iso20022records"
+version = "1.0.0"
+distribution = "2201.10.1"
+
+
```

Comment:
```suggestion
```

Reply:
Lets remove all necessary lines at the end of each files

Reply:
fixed in `f057766`